accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -540,6 +545,21 @@ all non-extension type members named `n` that _DV_ has, again using a
 repeated computation of the instantiated type of each superinterface on the
 path to the given non-extension type superinterface.
 
+There are two exceptions to these rules: _DV_ does not have a setter named
+`n=` when _DV_ declares a method named `n`, and _DV_ has a direct
+superinterface that has a setter named `n=`, and _DV_ does not declare a
+setter named `n=`. Similarly, _DV_ does not have a method named `n` when
```

Comment:
> The "DV does not declare a setter named n=" is unnecessary, that setter would conflict with the method named n and be a compile-time error before we even get to inheritance.

An earlier version of this PR did not have "DV does not declare a setter named n=", for exactly the reason you mention. Then I added it, because I wanted to describe the situation explicitly in so much detail that we can directly use the rules stated a few lines earlier to conclude that _DV_ _has_ a setter named `n=`, and then we proceed to say "but in this case _DV_ doesn't have a setter named `n=` after all".

The last bit gets murky if we deliberately allow for a _DV_ that declares a setter named `n=`. I just want to be able to say that "_DV_ doesn't have a member named `n=`", rather than "_DV_ doesn't have a member named `n=`, except if _DV_ declares a member named `n=`, but in that case we have another error". It's just simpler and more straightforward if we make sure from the beginning that _DV_ does not declare a member named `n=`.

I like the generalization! I'll work on that tomorrow.

Reply:
Used the generalization, the result is much better!